tests/formats/kzg/verify_blob_kzg_proof.md

```diff
@@ -0,0 +1,23 @@
+# Test format: Verify blob KZG proof
+
+Use the blob KZG proof to verify that the KZG commitment for a given `blob` is correct
+
+## Test case format
+
+The test data is declared in a `data.yaml` file:
+
+```yaml
+input:
+  blob: Blob -- the data blob
+  commitment: KZGCommitment -- the KZG commitment to the data blob
+  proof: KZGProof -- The KZG proof
+output: bool -- true (valid proof) or false (incorrect proof)
+```
+
+- `blob` here is encoded as a string: hexadecimal encoding of `4096 * 32 = 131072` bytes, prefixed with `0x`.
+
+All byte(s) fields are encoded as strings, hexadecimal encoding, prefixed with `0x`.
+
+## Condition
+
+The `verify_blob_kzg_proof` handler should verify that `commitment` is a correct KZG commitment to `blob` by using the blob KZG proof `proof`, and the result should match the expected `output`. If the commitment or proof are invalid (e.g. not on the curve or not in the G1 subgroup of the BLS curve) or `blob` is invalid (e.g. incorrect length or one of the 32 byte blocks does not represent a BLS field element), it should error, i.e. the output should be `None`.
```

Comment:
> If the commitment or proof is invalid (e.g. not on the curve or not in the G1 subgroup of the BLS curve) or `blob` is invalid (e.g. incorrect length or one of the 32-byte blocks does not represent a BLS field element), it should error, i.e. the output should be `null`.

Note that it's not as same as the pyspec BLS signature `verify_*` methods: in BLS signature API implementation, we catch the exception and return `False`. 

I'm not suggesting adding error handling in `polynomial-commitments.md`, I think that's too Pythonic. Just note that the clients have to deal with the error handling themselves.

btw, does `c-kzg` also throw exceptions in these cases? /cc @jtraglia

Reply:
I'm not entirely sure what cases you're referring to, but the C library in `c-kzg-4844` will return `C_KZG_BADARGS` if given invalid inputs. Generally, the bindings will throw an exception if the library doesn't return `C_KZG_OK` (success).

Reply:
For example, the case that the input is in the correct length but doesn't pass the subgroup check. I suppose it's not `C_KZG_OK` there so will throw an exception?

Reply:
Yes, the bindings will throw an exception. For example:

https://github.com/ethereum/c-kzg-4844/blob/ae0aeaf33d4433bc4c25e10bffc59cf7f1aa6650/bindings/java/c_kzg_4844_jni.c#L207-L211